src/Humanizer.Tests/ApiApprover/PublicApiApprovalTest.approve_public_api.approved.txt

```diff
@@ -1945,4 +1950,16 @@ namespace Humanizer.Localisation.Ordinalizers
         string Convert(int number, string numberString);
         string Convert(int number, string numberString, Humanizer.GrammaticalGender gender);
     }
+}
+namespace Humanizer.Localisation.TimeToClockNotation
+{
+    public enum ClockNotationRounding
```

Comment:
This enum should be in the root `Humanizer` namespace so that people don't need a deep include for this